README.md

```diff
@@ -189,3 +189,15 @@ fmt.Println(string(body))
 
 }
 ```
+
+```csharp
+
+// C#
+
+var client = new RestClient("http://api.pindo.io/v1/sms/");
```

Comment:
You should show how this `RestClient` is imported.

Reply:
RestSharp is core library within the C sharp lang no need to import.